infrastructure/modules/products-web/main.tf

```diff
@@ -1,10 +1,4 @@
 
-resource "azurerm_storage_container" "products_website" {
```

Comment:
are we sure we want to remove this `products_website`?

Reply:
Yeah, ever since we [added the static website block](https://github.com/MHRA/products/pull/571/files#diff-02147329094c80b8475c3f2e8ff56844) it looks like Azure creates the `$web` container on it's own. Will add it back if that turns out not to be the case